tests/Feature/PatientStoreDeviceTokenTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Enums\PlatformType;
+use App\Models\Doctor;
+use App\Models\Patient;
+use Carbon\Carbon;
+use Database\Seeders\DoctorSpecializationsSeeder;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Laravel\Passport\Passport;
+use Tests\TestCase;
+
+class PatientStoreDeviceTokenTest extends TestCase
+{
+    use RefreshDatabase;
+
+    public function setup(): void
+    {
+        parent::setUp();
+    }
+    
+    /** @test */
+    public function patient_successfully_store_token()
+    {
+        $patient = Patient::factory()->create(["verified_at" => Carbon::now()]);
```

Comment:
Please add tests to assert push notifications sent when adding or changing appointments status